test/integration/no_kubernetes_test.go

```diff
@@ -70,6 +71,17 @@ func TestNoKubernetes(t *testing.T) {
 	})
 }
 
+// ErrorStartNoK8sWithVersion expect an error when starting a minikube cluster without kubernetes and with a kubernetes version.
+func errorStartNoK8sWithVersion(ctx context.Context, t *testing.T, profile string) {
```

Comment:
simmilar to other helper functions, the name should validate..Something. so it can be used in the document generation

Reply:
Yes, applied suggestion.